tests/test_vec_normalize.py

```diff
@@ -35,13 +35,14 @@ def step(self, action):
         self.t += 1
         index = (self.t + self.return_reward_idx) % len(self.returned_rewards)
         returned_value = self.returned_rewards[index]
-        return np.array([returned_value]), returned_value, self.t == len(self.returned_rewards), {}
+        done = truncated = self.t == len(self.returned_rewards)
```

Comment:
same, we might want to test the influence of truncations on the statistics separetely?

Reply:
I need to think more about that one, but my guess is that it should not matter much, the scaling is meant to avoid large values and to evolve with the agent performance. Still, we should double check that.
Could you open an issue so we don't forget?